static/app/components/errors/detailedError.tsx

```diff
@@ -29,12 +30,12 @@ type Props = {
 
 function openFeedback(e: React.MouseEvent) {
   e.preventDefault();
-  Sentry.showReportDialog();
+  Sentry.showReportDialog({eventId: getLastEventId()});
```

Comment:
😬

Why are we removing lastEventId from the SDK?

Reply:
We relied on storing this on the hub, and figured instead of manually keeping this in module scope somewhere in the SDK the exact same thing can be done in user land. But I'm not sure, how important this use case is that we have here 🤔 cc @lforst , thoughts?

Reply:
Fair. We removed it because it didn't make sense with the new scope model but I can see people missing this API. The thing is, where should it live going forward? I guess the client or isolation scope make the most sense?

Reply:
I'd vote for client, I think?